network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -11,6 +11,14 @@
 from ConnectivityGraph import ConnectivityGraph
 from OutputConfiguration import OutputConfiguration
 from Peer import PeerSet, IpBlock
+from enum import Enum
```

Comment:
place this import with all the other standard imports (i.e., after `import itertools`)